MANUAL.md

```diff
@@ -262,6 +262,8 @@ The `mirroring` section lets you adjust mirroring behavior.
     to `false` if absent.
   * `mirroring.mirror_src`:
     Whether to mirror source (arch = `src`) RPM packages or not.
+  * `mirroring.revalidate_repodata`:
+    Whether to re-validate repodata content and packages when the repodata did not change
```

Comment:
It would be nice to have a statement about the default behavior when the key is absent in the configuration file (as in lines 261~262), as defined by the new method in `RMT::Config`:

```suggestion
    Whether to re-validate repodata content and packages when the repodata did not change. Defaults to `true` if absent.
```